usr/share/rear/lib/bootloader-functions.sh

```diff
@@ -620,18 +620,35 @@ function make_pxelinux_config_grub {
     # else set it based on PXE_TFTP_UR variable.
     if [[ -z $PXE_TFTP_IP ]] ; then
         if [[ -z $PXE_TFTP_URL ]] ; then
-            LogPrint "Can't find TFTP IP information. Variable TFTP_SERVER_IP or PXE_TFTP_URL with clear IP address must be set."
+            LogPrintError "Can't find TFTP IP information. Variable TFTP_SERVER_IP or PXE_TFTP_URL with clear IP address must be set."
             return
         else
             # Get IP address from PXE_TFTP_URL (ex:http://xx.yy.zz.aa:port/foo/bar)
             PXE_TFTP_IP=$(echo "$PXE_TFTP_URL" | awk -F'[/:]' '{ print $4 }')
+
+            # If PXE_TFTP_IP is not an IP, it could be a FQDM that must be resolved to IP.
+            # is_ip() function is defined in network-function.sh
+            if ! is_ip $PXE_TFTP_IP ; then
+
+                Log "Trying to resolve [$PXE_TFTP_IP] to a valid IP."
+                # get_ip_from_fqdn() function is defined in network-function.sh
+                PXE_TFTP_IP=$(get_ip_from_fqdn $PXE_TFTP_IP)
+
+                if ! is_ip $PXE_TFTP_IP ; then
```

Comment:
get_ip_from_fqdn already throws an error, will this be called whatsoever? IIRC `Error` will kill ReaR even if called from a subshell (thanks @jsmeix !). This if clause is therefore probably not required here. (I am just assuming, didn't test it).